---
title: "Members | Hibberd Lab"
layout: page
description: Hibberd Lab members past and present
---

{% include JB/setup %}

# Group members

Julian established his laboratory in the Department of Plant Sciences at the University of Cambridge in 2001.  Since then, nineteen post-
doctoral researchers, nineteen PhD students, and five research assistants have been part of the lab.  The group is currently seventeen-strong. 
Read more about them below.

----

<div class="row-fluid">
  <div class="span8 offset2 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/jh.jpg">
    <div class="media-body">
      <h4 class="media-heading">Julian Hibberd</h4>
      <p>Julian did his PhD with John Farrar and Bob Whitbread at Bangor University.  He then moved to post-doctoral positions with Julie 
      Scholes, Paul Quick and Malcolm Press in Sheffield, and then John Gray in Cambridge.  He started his own group in Cambridge while 
      holding a BBSRC Sir David Phillips Fellowship. His interests in plant biology have focussed on various aspects of photosynthesis.
      </p>
      <p>Julian has more recently become interested in using natural variation to improve our understanding of complex traits.  He is a 
      team leader within both the C<sub>4</sub> Rice Project, and represents the photosynthesis and metabolism community on the Plant 
      Section committee of the <a href="http://www.sebiology.org">Society of Experimental Biology</a>.  He is also an <a 
      href="http://www.plantphysiol.org/site/misc/edboard.xhtml">Associate Editor</a> of <a href="http://www.plantphysiol.org"><i>Plant 
      Physiology</i></a>, and from 2011-2014 was a member of the <a href="http://www.bbsrc.ac.uk/home/home.aspx">BBSRC</a> Responsive 
      Mode Grants panel.</p>  
    </div>
  </div>
</div>

----  

## Current members

### Post-Doctoral Research Associates

<div class="row-fluid">
    <div class="span6 media">
	<img class="pull-left media-object img-rounded" src="../assets/members/lh.jpg">
    <div class="media-body">
      <h4 class="media-heading">Lei Hua</h4>
      <p>Lei got his PhD in Crop Genetics and Breeding at China Agricultural University in 2015, during his PhD study, he identified and 
      characterized a domestication gene controlling long and barbed awns from the progenitor of cultivated rice, common wild rice (Oryza 
      rufipogon). Lei joined Hibberd lab in 2016; he is investigating the regulatory mechanisms of cell-specific gene expression in 
      C<sub>3</sub> leaves.</p>
    </div>
  </div>
  <div class="span6 media">
     <img class="pull-left media-object img-rounded" src="../assets/members/ts.jpg">
    <div class="media-body">
      <h4 class="media-heading">Tianshu Sun</h4>
      <p>Tianshu got her PhD in 2018 at Peking University, where she studied the genetic basis of adaptation to alpine environments in <i>Arabidopsis thaliana</i> populations in Tibet. Later, she worked as a research assistant at the Chinese Academy of Agricultural Science and focused on population genomics and transcriptomics in crop plants. She joined the Hibberd lab in November 2020 where she is currently investigating the regulation of photosynthesis gene expression variation at intra- and interspecific levels using genomics approaches such as eQTL-mapping and single-cell sequencing.</p>
    </div>
  </div>
 </div>
 <br>
 
<div class="row-fluid">
    <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/ps.jpg">
    <div class="media-body">
      <h4 class="media-heading">Pallavi Singh</h4>
      <p>Pallavi did a PhD in 2015 at National Institute of Plant Genome Research, India. She worked on investigating the role of Mitogen       Activated Protein Kinase (MAPK) signaling networks in imparting flooding tolerance in rice. In October 2015, she moved as a post-         doctoral associate to Cornell University, US, where her research interests focused on the interaction of rice with its bacterial         pathogen Xanthomonas and the role of transcription activator-like (TAL) effector proteins facilitating this interplay.
      In 2017 she started a post-doctoral position at the University of Cambridge, and is currently working on elucidating the spectrum         of regulatory networks governing the evolution of C<sub>4</sub> photosynthesis.</p> 
    </div>
  </div>
    <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/sst.jpg">
    <div class="media-body">
      <h4 class="media-heading">Sean Stevenson</h4>
      <p>Sean got his PhD from the University of Leeds where he studied abscisic acid (ABA) signalling in the model bryophyte                   Physcomitrella patens. This work identified a novel ABA regulator that was likely important in plant's conquest of land. After           leaving the north for warmer climes, Sean joined the Hibberd lab at the start of 2017. He will be using another bryophyte model           (Marchantia this time) to study light signalling and regulation and how this is integrated in early plant development and                 photosynthesis.</p> 
    </div>
  </div>
 </div>
<br>

<div class="row-fluid">
    <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/ne.jpg">
    <div class="media-body">
      <h4 class="media-heading">Natasha Elina</h4>
      <p>Natasha's aim is to understand the assembly of the photosynthetic apparatus and then manipulate it to induce chloroplast development in cells that are not normally green with the overarching aim to increase photosynthetic efficiency and crop productivity. Studies in Arabidopsis thaliana provide good understanding of how photosynthetic apparatus in assembled, however, this knowledge is not easily translated into grasses (including a staple monocot crop, rice). This is due to a number of whole genome duplication events and a whole genome triplication event that separate grasses from Arabidopsis. She is using a liverwort, Marchantia polymorpha, which, due to a low complexity of its genome, is an excellent tool to gain understanding of the basic ancestral pathways of the photosynthetic machinery development enabling translation of this knowledge into monocot crops.</p>
    </div>
  </div>
   <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/lc.jpg" >
    <div class="media-body">
      <h4 class="media-heading">Lee Cackett</h4>
      <p>Lee got her PhD in 2020 from the Molecular and Cellular Biology Department at the University of Cape Town, South Africa. During her PhD, she identified a role for auxin and auxin-related genes during the plant response to the ionic component of salinity stress. Lee joined the Hibberd lab in March 2020. Here she will be working to identify and characterise master transcriptional regulators of photosynthesis in rice and analyse the transcriptional changes in maize bundle sheath and mesophyll cells during growth in photoinhibitory conditions. </p>
    </div>
  </div>
 </div>
<br>

<div class="row-fluid">
    <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/pd.jpg">
    <div class="media-body">
      <h4 class="media-heading">Patrick Dickinson</h4>
      <p>Patrick did his PhD with Phil Wigge at the Sainsbury Laboratory Cambridge University working on how plants sense temperature and how temperature is integrated with other environmental signals to control adaptive responses in Arabidopsis. This work found a role for chloroplast signalling in controlling diurnal patterns of thermotolerance. Since moving across Cambridge to the Hibberd lab in summer 2017, Patrick is working on how cell-type sepcific expression is controlled in <i>Arabidopsis</i> and how large increases in the expression of genes required for C<sub>4</sub> photosynthesis have evolved using <i>Arabidopsis</i> and the C<sub>4</sub> model species <i>Gynandropsis gynandra</i>. </p>
    </div>
  </div>
   <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/ll.jpg">
    <div class="media-body">
      <h4 class="media-heading">Leonie Luginbuehl</h4>
      <p>Leonie did her PhD with Professor Giles Oldroyd at the John Innes Centre in Norwich, where she investigated the downstream transcriptional targets of three GRAS-domain transcription factors required for the establishment of the arbuscular mycorrhizal symbiosis in the model legume <i>Medicago truncatula</i>. Her work identified a lipid biosynthesis pathway that is upregulated in root cells colonized by arbuscular mycorrhizal fungi and provides the obligate biotrophic fungus with fatty acids as a major carbon source. In November 2017, Leonie joined the Hibberd lab as a postdoctoral scientist. Her research focuses on understanding the cell type specific regulation of photosynthetic capacity in rice leaves. </p>
      </div>
  </div>
 </div>
<br>

<div class="row-fluid">
    <div class="span6 media">
	  <img class="pull-left media-object img-rounded" src="../assets/members/kb.jpg" >
    <div class="media-body">
      <h4 class="media-heading">Kumari Billakurthi</h4>
      <p>Kumari got her PhD in December 2018 from Heinrich Heine University, Germany. She had performed EMS mutagenesis and activation tagging screens in Arabidopsis thaliana and had analysed comparative transcriptomics of leaf developmental gradients from C3 and C4 Flaveria species to identify novel genes that determine C4 leaf anatomy. She then continued in the same group as a postdoctoral researcher until the end of 2019. In April 2020, she has joined the Hibberdâ€™s group as a research associate to investigate how chloroplasts biogenesis is regulated by cell division patterns in rice (Oryza sativa) bundle sheath cells.</p>
      </div>
  </div>
  <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/tbs.jpg">
    <div class="media-body">
      <h4 class="media-heading">Tina B. Schreier</h4>
      <p>Tina did her PhD in plant biochemistry at ETH Zurich, and her main research interest lies in plant metabolism and photosynthesis. She joined the lab of Julian Hibberd in October 2018 as a Swiss National Science Foundation Research Fellow, studying cell-to-cell connectivity in leaves of the <i>Cleome</i> family and how this trait might have facilitated the evolution of C<sub>4</sub> photosynthesis. During her PhD in the lab of Professor Sam Zeeman, she discovered two chloroplast proteins that were originally thought to act as enzymes play other important non-canonical roles. This includes the plastidial NAD-dependent malate dehydrogenase, where the protein but not the enzymatic activity is required for proper chloroplast development. After her PhD, she worked at the John Innes Centre in the group of Professor Alison Smith, studying transgenic wheat lines with altered carbohydrate metabolism in the grain.</p>
    </div>
  </div>
 </div>
<br>

<div class="row-fluid">
  <div class="span6 media">
  <img class="pull-left media-object img-rounded" src="../assets/members/gr.jpg">
    <div class="media-body">
      <h4 class="media-heading">Greg Reeves</h4>
      <p> Greg has a BSc in Genetics and MSc in Horticulture both from New Mexico State University where he studied the inheritance of 
      disease resistance and spicy flavour in chilli peppers. He discovered a new gene that inhibits disease resistance, helped sequence 
      the chilli pepper genome and developed one of the hottest peppers in the world. Additionally, he helped identify novel genes 
      involved in photosynthesis from large-scale mutant screening in green algae at the Carnegie Institute for Science, Stanford. At 
      Cambridge, he completed his PhD investigating the genetic basis for C4 photosynthesis from natural variation among C4 plants and traditional gene 
      mapping approaches to identify which genes are involved in developing the C<sub>4</sub> pathway. </p>
	 </div>
  </div>
  <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/EF_edit .jpg">
    <div class="media-body">
      <h4 class="media-heading">Eftychios Frangedakis</h4>
      <p>Eftychis did his PhD with Jane Langdale at the University of Oxford, working on plant evo-devo. He joined the Hibberd lab in March 2020. Here he is exploring synthetic biology approaches to increase the chloroplast compartment in Arabidopsis bundle sheath cells.</p>
    </div>
  </div>
 </div>
 <br>
  </div>
  <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/CC.jpg">
    <div class="media-body">
      <h4 class="media-heading">Caijin Chen</h4>
      <p>Caijin got her PhD in plant molecular genetics in 2021 in University of Aberdeen, UK, where she worked on genome-wide association mapping of salt tolerance and arsenic accumulation in rice. Caijin joined the Hibberd Lab in September 2021, she will be working to increase chloroplast content of rice bundle sheath cells to resemble more closely the bundle sheath of C4 leaves.</p>
    </div>
  </div>
 </div>
 <br>


### PhD students

<div class="row-fluid">
  <div class="span6 media">
  <img class="pull-left media-object img-rounded" src="../assets/members/cs.jpg">
    <div class="media-body">
      <h4 class="media-heading">Conor Simpson</h4>
      <p>Conor is a BBSRC DTP student. He graduated with an MSci from the University of Nottingham. For his masters project at Nottingham he worked on the N-end rule pathway of programmed protein degradation. Currently, through exploiting the natural variation in the C<sub>4</sub> species
      <i>Gynanropsis gynandra</i>, Conor is using specific breeding methods in conjunction with quantitative genetics to calculate the heritability
      and number of genes responsible for certain C<sub>4</sub> characteristics. Additionally, with the aim of identifying loci responsible for complex
      C<sub>4</sub> phenotypes, such as Kranz anatomy, he is using quantitative trait loci (QTL) analysis to generate a high-density genetic map in <i>G.
      gynandra</i>. </p>
    </div>
  </div>
  <div class="span6 media">
  <img class="pull-left media-object img-rounded" src="../assets/members/zad.jpg">
    <div class="media-body">
      <h4 class="media-heading">Zheng-Ao Di</h4>
      <p> Zhengao got a BSc in Chemistry from Nankai University, China, in 2016. After graduation, he took an intern in Shanghai Institute of Plant Physiology & Ecology for 6 months investigating the functional targets of secondary metabolites by chemical proteomics. Zhengao joined the Hibberd lab in Sep 2017 as a PhD student. He is now studying how modifications to the spatial organisation of metabolic pathways can be used to manipulate their efficiency in two complementary strands of research. In the first, he aims to better understand how assembly of synthetic protein scaffolds affects pathway efficiency in plants. In the second, he aims to increase the distance between enzymes of photosynthesis by engineering a synthetic C<sub>4</sub> cycle in leaves of C<sub>3</sub> <i>A. thaliana</i>.</p>
  </div>
  </div>
 </div>
<br>

### Masters Students

### Research staff

<div class="row-fluid">
  <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/ss.jpg">
      <div class="media-body">
      <h4 class="media-heading">Susan Stanley</h4>
      <p> Susan is in charge of keeping the lab running! She has been instrumental in developing the transformation pipeline that
      we use for <i>Gynandropsis gynandra</i>. She is also responsible for plant and seed stock maintenance, ordering lab stocks, and
      time permiting, helps with specific research projects in the lab. She joined the lab in 2000 after having worked in
      Industry.</p>
    </div>
    </div>
  <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/nw.jpg">
    <div class="media-body">
    <h4 class="media-heading">Na Wang</h4>
    <p>Na did her MSc in Plant Genetics and Breeding at CAU (China Agricultural University), then she went to Biotech center of Beijing 
  Da Bei Nong Technology Group Co. Ltd., she got 5 year rich experience of cereal genetic transformation there. She joined the Hibberd 
  lab in 2017 as a research technician.</p>
    </div>
    </div>
</div>

<div class="row-fluid">
    <div class="span6 media">
      <img class="pull-left media-object img-rounded" src="../assets/members/rd.jpg">
      <div class="media-body">
      <h4 class="media-heading">Ruth Donald</h4>
	  <p>Ruth has a PhD in Plant Biochemistry from the University of Warwick where she studied the targeting of chloroplast proteins into 
	  the thylakoid membrane and found that multiple pathways exist.  She then went to a postdoctoral post at the University of Auckland, 
	  before coming to Plant Sciences Cambridge work on the import, assembly and turn-over of chloroplast proteins in John Grayâ€™s lab, and 
	  then as a Royal Society Research Fellow. Since then she has worked at Biogemma on useful oil production in oilseed rape and screening 
	  of wheat cultivars for marker-gene trait associations, and has also worked for Syngene.  She returned to Plant Sciences in May 2017, 
	  joining the Hibberd lab to work on crop transformation.</p>
    </div>
   </div>
 <div class="span6 media">
    <img class="pull-left media-object img-rounded" src="../assets/members/at.jpg">
    <div class="media-body">
    <h4 class="media-heading">Anoop Tripathi</h4>
    <p>Anoop did his MSc in Biochemistry at University of Lucknow, India. Then he worked as Senior Research Fellow at National Bureau of Plant Genetic Resources (ICAR-NBPGR), New Delhi, India, where his research focused on understanding the gene regulatory networks in cereal crops. He joined the Hibberd lab as a Senior Research Technician in January 2020 and is currently working on grafting and somatic hybridisation.</p>
    </div>
    </div>
</div>

## Past members
#### Former PhD students
<div>- Britta KÃ¼mpers: Analysis of C<sub>4</sub> photosynthesis using Flaveria </div>
<div>- Richard Smith: Computational analysis of C<sub>4</sub></div>
<div>- Chris John: Comparative transcriptomics of C<sub>4</sub></div>
<div>- Helen Woodfield: Engineering C<sub>4</sub> genes into C<sub>3</sub> species</div>
<div>- Ben Williams: Mechanisms underlying the coordinated evolution of multiple C<sub>4</sub> genes</div>
<div>- Maria Herse: Crop improvement using synthetic variation</div>
<div>- Ewan Gage: The role of small RNAs in C<sub>4</sub> photosynthesis</div>
<div>- Ben Tolley: Oat maize addition lines</div>
<div>- Kaisa Kajala: Regulation of gene expression in C<sub>4</sub> plants</div>
<div>- Holly Astley: The role of PPDK regulatory proteins in C<sub>3</sub> plants</div>
<div>- Anna Leiss: The function of PEPCK in C<sub>3</sub> plants</div>
<div>- Lucy Taylor: The role of PPDK in nitrogen remobilisation</div>
<div>- Sophie Janacek: Understanding the role of photosynthesis in veins of C<sub>3</sub> plants</div>
<div>- Diana Marshall: Using Cleome to understand C<sub>4</sub> photosynthesis</div>
<div>- Meredith Wilson: Genome structure and function in parasitic angiosperms</div>
<div>- Jana Knerova: BS cell-specific expression of C<sub>3</sub> and C<sub>4</sub> plants</div>
<div>- Robyn Phillips: Activating photosynthesis in non-photosynthetic cells for improved crop productivity</div>
<br>

#### Former Post-Doctoral Research Associates
<div>- Dr. Sarah Covshoff: C<sub>4</sub> rice </div>
<div>- Dr. Sylvain Aubry: Molecular signature of the C<sub>4</sub> phenotype </div>
<div>- Dr. Naomi Brown: Role of C<sub>4</sub> acid decarboxylases</div>
<div>- Dr. Kate Parsley: Role and regulation of PPDK</div>
<div>- Dr. Alex Johnson: Rice enhancer traps</div>
<div>- Dr. Zheng Liu: Developing Cleome as a model</div>
<div>- Dr. Elisabeth Truernit: Chloroplast isolation from specific cell types</div>
<div>- Dr. Steven Burgess: Gene regulation of C<sub>4</sub> photosynthesis in Cleome</div>
<div>- Dr. Marek Szecowka: Using artificial scaffolds to increase photosynthetic efficiency in C<sub>3</sub> plants</div>
<div>- Dr. Ming-Tsung Louis Wu: Large-scale modeling analysis of C4 photosynthesis using de novo transcriptomics.</div>
<div>- Dr. Andy Plackett: Understanding differential regulation of chloroplast development between mesophyll and bundle sheath cells in rice (Oryza sativa).</div>
<div>- Dr. Ahmed Ismail: Understanding the genetic basis of C<sub>4</sub> photosynthesis during leaf development in rice and sorghum. Read more <a href="http://hibberdlab.com/members/ahmed">here</a> </div>
<div>- Dr. Ivan Reyna-Llorens: Transcriptional regulations of the C<sub>4</sub> pathway </div>
<div>- Dr. Sherif El-Sharnouby </div> 
<div>- Dr. Haiyan Xiong: The regulatory mechanism of light stress response in rice bundle sheath cells </div>
<br> 

#### Former Research Staff
<div>- Mathieu GrangÃ©-Guerment: C<sub>4</sub> rice</div>
<div>- Chris Boursnell: Wild Rice MAGIC project and <i>de novo</i> transcriptome assembly</div>
<br> 
