---
layout: page
title: "Projects"
description: "Projects"
---
{% include JB/setup %}

# Current research focus

Our work on understanding the C<sub>4</sub> pathway makes use of model organisms as well as natural variation, and classical molecular biology approaches as well as emerging technologies in bioinformatics and computational processing.

----

<!-- carousel -->
<div id="myCarousel" class="carousel slide">
  <div class="carousel-inner">
    <div class="item active">
      <img src="../assets/projects/evolution.png" alt="">
      <div class="container">
        <div class="carousel-caption">
          <h1>Comparative analysis of C<sub>3</sub> and C<sub>4</sub> gene expression</h1>
          <p class="lead">We are extending the phylogenetic breadth of RNAseq on closely related C<sub>3</sub> and C<sub>4</sub> species. This includes cell specific transcriptomes and will form the foundation for an understanding of the extent to which the M and BS transcriptomes of independent C<sub>4</sub> lineages are convergent.</p>
          <a class="btn btn-info" href="#comparative">Read more...</a>
        </div>
      </div>
    </div>
    <div class="item">
      <img src="../assets/projects/evolution2.png" alt="">
      <div class="container">
        <div class="carousel-caption">
          <h1>The ancestral role of proteins used in C<sub>4</sub> photosynthesis</h1>
          <p class="lead">The model C<sub>3</sub> plant <i>A. thaliana</i> contains all the proteins used in the C<sub>4</sub> pathway. We are using Arabidopsis to define the ancestral role of these proteins in C<sub>3</sub> species prior to their recruitment into the C<sub>4</sub> pathway.</p>
          <a class="btn btn-info" href="#ancestral">Read more...</a>
        </div>
      </div>
    </div>
    <div class="item">
      <img src="../assets/projects/c4_rice_project.png" alt="">
      <div class="container">
        <div class="carousel-caption">
          <h1>The C<sub>4</sub> rice project</h1>
          <p class="lead">We are involved in building the basic biochemistry required for the C<sub>4</sub> pathway in leaves of rice, this is done in collaboration with <a href="http://photosynthome.irri.org/C4rice/index.php/">IRRI</a>. In addition, in we have identified candidate regulators of the patterns of C<sub>4</sub> gene expression and with Bob Furbank (CSIRO) are testing the role of these proteins in <i>Setaria italica</i>.</p>
          <a class="btn btn-info" href="#c4rice">Read more...</a>
        </div>
      </div>
    </div>
    <div class="item">
      <img src="../assets/projects/rubisco.png" alt="">
      <div class="container">
        <div class="carousel-caption">
          <h1>Using synthetic approaches to increase CO<sub>2</sub> supply to RuBisCO</h1>
          <p class="lead">In collaboration with <a href="http://www.psrg.org.uk/">Mike Blatt, Nigel Burroughs, Cheryl Kerfield, Nick Smirnoff and John Golbeck</a> we are investigating whether photosynthesis can be enhanced using artificial protein scaffolds and light driven HCO<sub>3</sub><sup>-</sup> pumps.</p>
          <a class="btn btn-info" href="#rubiso">Read more...</a>
        </div>
      </div>
    </div>
    <div class="item">
      <img src="../assets/projects/wild_rices.png" alt="">
      <div class="container">
        <div class="carousel-caption">
          <h1>Overcoming the domestication bottleneck in rice</h1>
          <p class="lead">As with all major crops, rice went through a series of domestication bottlenecks
    that reduced allelic variation compared with its ancestors and current wild species.
    With <a href="http://photosynthome.irri.org/C4rice/index.php/">IRRI</a> we are using wild species of rice and Multi Advanced Generation Inter-Crossing to re-introduce tolerance to biotic and abiotic stresses.</p>
          <a class="btn btn-info" href="#wildrice">Read more...</a>
        </div>
      </div>
    </div>
  </div>
  <a class="left carousel-control" href="#myCarousel" data-slide="prev">&lsaquo;</a>
  <a class="right carousel-control" href="#myCarousel" data-slide="next">&rsaquo;</a>
</div>
<!-- end of carousel -->

<div class="well media">
  <img class="media-object pull-left img-rounded" src="../assets/projects/evolutio_small.png" style="width:150px; height:150px;">
  <a name='comparative' style='position:relative; top:-75px;'>&nbsp;</a>
  <div class="media-body">
    <h3 class="media-heading">Comparative analysis of C<sub>3</sub> and C<sub>4</sub> gene expression</h3>

    <p>With Andreas Weberâ€™s lab we initially conducted analysis of RNA populations in whole leaves of C<sub>3</sub> and C<sub>4</sub> <i>Cleome</i> species. We are extending this to undertake large-scale analysis of many more lineages, of leaf maturation, and of mesophyll and bundle sheath transcriptomes. To fully understand these large dataset, significant investment in development of computational approaches is required!</p>

   <p><i>Images: top and left: traced leaf venation patterns of C<sub>3</sub> vs. C<sub>4</sub> plants.</i></p>
  </div>
</div>

<div class="well media">
  <img class="media-object pull-left img-rounded" src="../assets/projects/evolution2_small.png" style="width:150px; height:150px;">
  <a name='ancestral' style='position:relative; top:-75px;'>&nbsp;</a>
  <div class="media-body">
    <h3>The ancestral role of proteins used in C<sub>4</sub> photosynthesis</h3>

    <p>Despite having well-characterised functions in C<sub>4</sub> species, many of the core enzymes of the C<sub>4</sub> pathway have poorly understood roles in C3 plants. We are analyzing the roles that they play through analysis of when and where they are expressed, as well as the use of insertional mutants and overexpression, followed by in depth photosynthetic and metabolic phenotyping. Many of these proteins are members of multi-gene families, and belong to redundant networks, and so the analysis is often intellectually challenging.</p>

    <p><i>Images: top and left: developing Arabidopsis seedlings expressing a marker</i></p>
  </div>
</div>

<div class="well media">
  <img class="media-object pull-left img-rounded" src="../assets/projects/rice_thumb.jpg" style="width:150px; height:150px;">
  <a name='c4rice' style='position:relative; top:-75px;'>&nbsp;</a>
  <div class="media-body">
    <h3>The C<sub>4</sub> rice project</h3>

    <p>Our focus has been generating resources that allow us to test the extent to which the basic C<sub>4</sub> cycle can be installed in rice. This has involved analysis of the extent to which C<sub>4</sub> genes from maize are expressed in a cell specific manner in rice, and also investigating potential regulators of these genes. Clearly, this is a long-term, incredibly challenging project, but we are finding that we learn a lot about the fundamentals of C<sub>4</sub> photosynthesis along the way.</p>

   <p><i>Images: top: C<sub>3</sub> rice (foreground) growing alongside C<sub>4</sub> Echinochloa glabrescens (mid) and maize (rear). The increased efficiency of C<sub>4</sub> enables the C<sub>4</sub> plants to grow faster with the same resources. left: rice growing in paddy conditions.</i></p>
  </div>
</div>

<div class="well media">
  <img class="media-object pull-left img-rounded" src="../assets/projects/rubisco_small.png" style="width:150px; height:150px;">
  <a name='rubisco' style='position:relative; top:-75px;'>&nbsp;</a>
  <div class="media-body">
    <h3>Using synthetic approaches to increase CO<sub>2</sub> supply to RuBisCO</h3>

    <p>For many decades, work has been conducted that aims to increase the efficiency of RuBisCO by modifying its catalytic characteristics. We are taking an alternate approach inspired by Synthetic Biology in order to investigate whether artificial protein scaffolds can be used to improve the efficiency of photosynthesis. We hope that we can use this as proof of concept for engineering plant metabolism in general.</p>

    <p><i>Images: top: structure of RuBisCO complex consisting of 8 large and and 8 small subunits. left: RuBP binding the active site of the RuBisCO large subunit.</i></p>
  </div>
</div>
<div class="well media">
  <img class="media-object pull-left img-rounded" src="../assets/projects/wild_rice.jpg" style="width:150px; height:150px;">
  <a name='wildrice' style='position:relative; top:-75px;'>&nbsp;</a>
  <div class="media-body">
    <h3>Overcoming the domestication bottleneck in rice</h3>

    <p>A significant amount of work has been undertaken in which individual wild relatives of rice have been used as sources for useful traits in domesticated rice. In this project we are broadening this concept to source variation present in all the AA rice genomes, and increase it further using the multi advance generation intercrossing approach. Back crossing into <i>O. sativa</i> allows useful traits to transferred into elite germplasm. The wide crossing, and the subsequent analysis of gene expression and genomic structures represent significant technical and computational challenges.</p>

    <p><i>Images: top and left: wild relatives of rice showing the broad morphological spectrum in the genus. This project aims to harness that diversity to improve cultivated rice.</i></p>
  </div>
</div>

## Past projects and funding

- Cleome as a model for C<sub>4</sub> photosynthesis (The Leverhulme Trust)
- The role and regulation of PPDK in C<sub>3</sub> species (BBSRC Responsive mode)
- The role of C<sub>4</sub> acid decarboxylases in Arabidopsis (BBSRC Responsive mode)
- Photosynthesis in discrete cell types. (BBSRC Sir David Phillips Research Fellowship)

Collaborations:

- With Dr Alex Webb. The circadian clock and photosynthesis. (BBSRC Responsive mode)
- With Dr Mark Tester and Emmanuel Guiderdoni. Rice enhancer traps (BBSRC Responsive mode)
